contracts/utils/contracts/src/v08/LibSafeMathV08.sol

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2020 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.8;
+
+import "./errors/LibRichErrorsV08.sol";
+import "./errors/LibSafeMathRichErrorsV08.sol";
+
+library LibSafeMathV08 {
+    function safeMul(uint256 a, uint256 b) internal pure returns (uint256) {
+        if (a == 0) {
+            return 0;
+        }
+        uint256 c = a * b;
+        if (c / a != b) {
```

Comment:
I'm only leaving this comment once, because this is present several times in this file.

Because there's no `unchecked` here, this `if` statement is vacuous and the `LibRichErrorsV08.rrevert` is unreachable. The built-in safemath will revert on `*`.